docker/startup.sh

```diff
@@ -45,4 +45,23 @@ if [ -n "$R_PACKAGES" ]; then
   R -e "install.packages($R_PACKAGES, repos = 'http://cran.us.r-project.org')"
 fi
 
+# Install custom cacerts
+if [ -e /mnt/mesos/sandbox/cacerts ]; then
```

Comment:
`$MESOS_SANDBOX` should always be used instead of directly hard-coding the path `/mnt/mesos/sandbox`